### DISCUSSION TITLE
What is the difference between FOSS and non-FOSS version?

### DISCUSSION THREAD

**Discussion by Aryan-Chauhan** [T8RIN/ImageToolbox#1322] [Q&A] [answered]
Can somebody tell me the exact differences between the two?

![Screenshot_2024-09-06-10-45-36-15_320a9a695de7cdce83ed5281148d6f19.jpg](https://github.com/user-attachments/assets/8e146287-5230-4d12-bd51-4ea745843646)

Thank you.

**Comment by NikunjK8** [comment]
Foss versions will have less features, idk what features exactly

**Comment by ksharizard** [comment, marked as answer]
https://github.com/T8RIN/ImageToolbox?tab=readme-ov-file#%EF%B8%8F-foss-vs-market
